sns/parity_math.c

```diff
@@ -1900,7 +1929,7 @@ static int ir_recover(struct m0_sns_ir *ir, struct m0_sns_ir_block *alive_block)
 
 		failed_bufvec = ir->si_blocks[ir->si_failed_idx[i]].sib_addr;
 		/* Get data from failed vector in buffer */
-		bufvec_buf(failed_bufvec, &buf, 1, true);
+		bufvec_to_buf_copy(failed_bufvec, &buf, 1);
```

Comment:
Actually, in current code, the arguments of this function is just a one-element array.
So, we can treat the arguments as a pointer to a single struct.

Reply:
Updated bufvec_to_buf_copy() function.